docs/SRS-Volere/SRS.tex

```diff
@@ -706,8 +706,11 @@ \section*{Appendix --- Reflection}
 
   \medskip
   \textbf{Sawyer Tang}\\
-  \textit{Chosen Skills}: \\
-  \lips
+  \textit{Chosen Skills: (a), (b), (d), (h)} \\
+  For this project, I was assigned to backend development. In following this, I am choosing to improve my abilities in \textit{NodeJS} and \textit{SQL}. I have used some of these technologies in brevity over my past internship positions but I am looking to deepen my experience with them further with the hopes of bringing this experience to future personal project and jobs. 
+  In addition to wanting to improve my backend skills, I also want to dive into front end. More specifically, mobile development in \textit{Flutter}. This is something that I have always had an interest in learning and this project seems to me like a perfect opportunity.
+  Finally, I believe that being able to effectively communicate your ideas is an invaluable skill and it is something that I am always looking to improve for myself and career. \par
+  My approach to gaining these frontend and backend skills will be to consume public documentation, online forums, and \textit{YouTube} video tutorials. Problem-solving with my peers is also a useful and fun method of improving technical skills which I hope to use during this project. For communication skills, I practice them every day during meetings or when working with team members. I also look to improve these skills during our team presentations and demos.
```

Comment:
I think the intention is to stick to the approaches that I mentioned in the preamble, so maybe online articles instead of forums for the first sentence. I didn't mention problem solving for the technical skills at the beginning (in regards to the second sentence) but I did mention open source projects as reference.